app.rb

```diff
@@ -0,0 +1,188 @@
+require_relative './classroom'
+require_relative './book'
+require_relative './rental'
+require_relative './person'
+require_relative './teacher'
+require_relative './student'
+
+# App Class that will serve as your console app entry-point.
+class App
+  def initialize
+    @books_arr = []
+    @people_arr = []
+    @rentals_arr = []
+  end
+
+  def book_store(book)
+    @books_arr.push(book)
+    @books_arr
+  end
+
+  def people_store(person)
+    @people_arr.push(person)
+    @people_arr
+  end
+
+  def rentals_store(rental)
+    @rentals_arr.push(rental)
+    @rentals_arr
+  end
+
+  def user_input
+    puts 'Welcome to School Library App'
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts '7 - Exit'
+  end
+
+  def list_books
+    @books_arr.each do |book|
+      puts "Title: #{book.title}, Author: #{book.author}"
+    end
+    main
+  end
+
+  def list_people
+    @people_arr.each { |person| puts "[#{person.class}]: Name: #{person.name}, ID: #{person.id} AGE: #{person.age}" }
+    main
+  end
+
+  def choice_permission
+    choice = gets.chomp.downcase
+    permission = true
+    case choice
+    when 'y'
+      permission = true
+    when 'n'
+      permission = false
+    else
+      puts 'Invalid Submission'
+    end
+    [permission]
+  end
+
+  def create_teacher
+    puts 'What subject does the teacher take?'
+    subject = gets.chomp
+    puts 'What is the age of the teacher?'
+    age = gets.chomp.to_i
+    puts 'What is the name of the teacher?'
+    name = gets.chomp
+    puts 'Does the teacher have permission to use the library services [Y/N]: ?'
+    permission = choice_permission
+    teacher = Teacher.new(subject, age, name, parent_permission: permission)
+    people_store(teacher)
+  end
```

Comment:
- [ ] When I am creating a person 1 should be a `student` and 2 should be a `teacher` currently when I press 1 it is displays questions of the teacher and vice versa please address that
- [ ] Remember when  you are creating a teacher  you will first ask his/her `specialization` then `name`, `age`